src/Microsoft.ML.StandardLearners/Standard/MultiClass/Pkpd.cs

```diff
@@ -68,94 +67,161 @@ public sealed class Pkpd : MetaMulticlassTrainer<PkpdPredictor, Pkpd.Arguments>
         public sealed class Arguments : ArgumentsBase
         {
         }
-
+        /// <summary>
+        /// Legacy constructor that builds the <see cref="Pkpd"/> trainer supplying the base trainer to use, for the classification task
+        /// through the <see cref="Arguments"/>arguments.
+        /// Developers should instantiate <see cref="Pkpd"/> by supplying the trainer argument directly to the <see cref="Pkpd"/> constructor
+        /// using the other public constructor.
+        /// </summary>
+        /// <param name="env"></param>
+        /// <param name="args"></param>
         public Pkpd(IHostEnvironment env, Arguments args)
             : base(env, args, LoadNameValue)
         {
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Pkpd"/>
+        /// </summary>
+        /// <param name="env">The <see cref="IHostEnvironment"/> instance.</param>
+        /// <param name="singleEstimator">An instance of the <see cref="BinaryPredictionTransformer{IModel}"/> used as the base predictor.</param>
```

Comment:
>singleEstimator [](start = 25, length = 15)

the comment is incorrect.

Reply:
this is a trainer, not a transformer

---
In reply to: [217233766](https://github.com/dotnet/machinelearning/pull/865#discussion_r217233766) [](ancestors = 217233766)